src/ffi/c.rs

```diff
@@ -447,7 +447,7 @@ mod c_backend {
     #[cfg(feature = "zlib-ng")]
     const ZLIB_VERSION: &'static str = "2.1.0.devel\0";
     #[cfg(all(not(feature = "zlib-ng"), feature = "zlib-rs"))]
-    const ZLIB_VERSION: &'static str = "1.3.0-zlib-rs-0.3.0\0";
```

Comment:
this bump is not strictly needed here (only the `1.3.0` part is relevant, and I believe only the `1` prefix is checked by most zlib libraries (but applications might check more).